src/backends/tensorflow/base_backend.cc

```diff
@@ -609,7 +609,7 @@ BaseBackend::Context::SetStringInputTensor(
       content_byte_size -= len;
 
       TRTISTF_TensorSetString(
-          tensor, tensor_element_idx + element_idx, str.c_str());
+          tensor, tensor_element_idx + element_idx, str.c_str(), len);
```

Comment:
Don't need 'str' anymore, just use "content, len" here.
And move lines 608 and 609 to be just before 613

Reply:
Done